lib/cc/analyzer/issue.rb

```diff
@@ -24,7 +24,7 @@ def as_json(*)
       end
 
       def fingerprint
-        parsed_output.fetch("fingerprint", default_fingerprint)
+        parsed_output["fingerprint"] || default_fingerprint
```

Comment:
I went into more detail in the commit message. This is more about not going through the trouble of reading a file / computing the default fingerprint if we already have one in the issue output.